<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QWidget" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/Start.svg</normaloff>:/icons/Start.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: #172027;
color:#E6E6E6; 
border: 1px rgb(67, 67, 67);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="coreApps">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 1px rgb(67, 67, 67);
padding: 5px 5px 5px 5px;
}

QPushButton:checked{
color: #DCDCDC; 
font-weight: bold;
}

QPushButton:!checked{
color: #57585D ;
}</string>
        </property>
        <property name="text">
         <string>CoreApps</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="speedDial">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 1px rgb(67, 67, 67);
padding: 5px 5px 5px 5px;
font-size: 22px;
}

QPushButton:checked{
color: #DCDCDC; 
font-weight: bold;
}

QPushButton:!checked{
color: #57585D ;
}</string>
        </property>
        <property name="text">
         <string>Speed Dial</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recentActivites">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: 1px rgb(67, 67, 67);
padding: 5px 5px 5px 5px;
font-size: 22px;
}

QPushButton:checked{
color: #DCDCDC; 
font-weight: bold;
}

QPushButton:!checked{
color: #57585D ;
}</string>
        </property>
        <property name="text">
         <string>Recent Activites</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="pages">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="pcoreApps">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>25</number>
       </property>
       <item>
        <widget class="QListWidget" name="appCollect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListView::item:selected {
border: 1px solid #232730;
background-color: #232730;
}

QListView::item:hover {
background-color: #232730;
}

QListView::item {
background-color: #303942;
border: 1px #323232;
padding: 5px 20px 5px 20px;
border-radius: 2px;
}

QListView{
padding: 50px 5px 20px 20px;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::IgnoreAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="movement">
          <enum>QListView::Snap</enum>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="layoutMode">
          <enum>QListView::SinglePass</enum>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="gridSize">
          <size>
           <width>130</width>
           <height>113</height>
          </size>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <property name="batchSize">
          <number>10</number>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="selectionRectVisible">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>CoreBox</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/Box.svg</normaloff>:/icons/Box.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CoreTerminal</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/CoreTerminal.svg</normaloff>:/icons/CoreTerminal.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CorePDF</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/CorePDF.svg</normaloff>:/icons/CorePDF.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CoreFM</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/CoreFM.svg</normaloff>:/icons/CoreFM.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CorePad</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/CorePad.svg</normaloff>:/icons/CorePad.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CorePaint</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/CorePaint.svg</normaloff>:/icons/CorePaint.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CorePlayer</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/CorePlayer.svg</normaloff>:/icons/CorePlayer.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CoreImage</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/CoreImage.svg</normaloff>:/icons/CoreImage.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Search</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/Search.svg</normaloff>:/icons/Search.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DashBoard</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/DashBoard.svg</normaloff>:/icons/DashBoard.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bookmarks</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/Bookmarks.svg</normaloff>:/icons/Bookmarks.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Settings</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/Settings.svg</normaloff>:/icons/Settings.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>About</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/About.svg</normaloff>:/icons/About.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Help</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/Help.svg</normaloff>:/icons/Help.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CoreTime</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/CoreTime.svg</normaloff>:/icons/CoreTime.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CoreRenamer</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/CoreRenemer.svg</normaloff>:/icons/CoreRenemer.svg</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pspeedDial">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>25</number>
       </property>
       <item>
        <widget class="QListWidget" name="speedDialB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>450</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListView::item:selected {
border: 1px solid #232730;
background-color: #232730;
}

QListView::item:hover {
background-color: #232730;
}

QListView::item {
background-color: #303942;
border: 1px #323232;
padding: 5px 20px 5px 20px;
border-radius: 2px;
}

QListView{
padding: 50px 5px 20px 20px;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::IgnoreAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="movement">
          <enum>QListView::Snap</enum>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="gridSize">
          <size>
           <width>130</width>
           <height>113</height>
          </size>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <property name="batchSize">
          <number>10</number>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="selectionRectVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="precents">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>45</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
